include/UIKit/UIImage.h

```diff
@@ -16,14 +16,18 @@
  *    be used to endorse or promote products derived from this software without
  *    specific prior written permission.
  *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
```

Comment:
hmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm

---
In reply to: [84551045](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84551045) [](ancestors = 84551045)